src/main/java/com/chat/yourway/controller/ContactController.java

```diff
@@ -22,10 +23,23 @@ public class ContactController {
 
     @Operation(summary = "Change to new password")
     @PatchMapping(path = "/password",
-            produces = APPLICATION_JSON_VALUE,
             consumes = APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.NO_CONTENT)
     public void changePassword(@RequestBody ChangePasswordDto request, @AuthenticationPrincipal UserDetails userDetails) {
         contactService.changePassword(request, userDetails);
     }
+
+    @Operation(summary = "Send email to restore password")
+    @PostMapping(path = "/password/email",
+            consumes = APPLICATION_JSON_VALUE)
+    public void sendRequestToRestorePassword(@RequestBody String email, HttpServletRequest httpRequest) {
```

Comment:
Do we really need to use @RequestBody for single String email? why can`t we use @RequestParam